spring-cloud-azure-starters/spring-cloud-azure-starter-storage/README.md

```diff
@@ -0,0 +1,66 @@
+# Spring Cloud Azure Storage 
+
+The project implements Spring Resource abstraction for Azure Storage which allows you to interact with Azure Storage instances using Spring programming model.
+
+## Feature List 
+
+- [Auto-configuration for Azure Storage](#auto-configuration-for-azure-storage) 
+- [Read a file from Azure Storage](#read-a-file-from-azure-storage)
```

Comment:
Read a resource. Resource cloud be a single file or a collection of files. We might support more.

Reply:
For Azure Storage blob.